pkg/server/server.go

```diff
@@ -2245,25 +2245,30 @@ func (s *topLevelServer) runIdempontentSQLForInitType(
 	}
 
 	initAttempt := func() error {
-		const defaulVirtuallusterName = "main"
+		const defaulVirtualClusterName = "main"
```

Comment:
super nit: `defaulVirtualClusterName` -> `defaultVirtualClusterName`

Reply:
Ha! I really can't believe I spotted one typo and not the other. I'll fix this on master first.